app/assets/javascripts/hated-foods.js

```diff
@@ -18,8 +16,43 @@ $(document).ready(function(){
     });
   };
 
-   $('.hated-food').children().on('click', function(){
+  $('.hated-food').find('button').on('click', function(){
     $(this).toggleClass('btn-danger');
   });
 
+  $("#favorite-number").bind('click', function(){
+    $("#favorite-number-selection").toggleClass('hidden')
+  });
+
+  $("#specific-food").on("ajax:success", function(e, data, status, xhr) {
```

Comment:
Mixed double and single quotes.<br>'xhr' is defined but never used.<br>'status' is defined but never used.